src/search/pages/Search/Search.tsx

```diff
@@ -1,59 +1,59 @@
-import { useCallback, useRef } from 'react';
+import { Suspense } from 'react';
 
 import { useSearchParams } from 'react-router-dom';
 
 import Spacing from '@/components/Spacing/Spacing';
+import { ErrorBoundary } from '@/search/components/ErrorBoundary/ErrorBoundary';
+import {
+  ResultListFallbackComponent,
+  TotalFallbackComponent,
+} from '@/search/components/FallbackComponent/FallbackComponent';
+import { RealTimeKeyword } from '@/search/components/RealTimeKeyword/RealTimeKeyword';
+import { ResultListItemLoading } from '@/search/components/ResultListItem/ResultListItemLoading';
+import { ResultListItems } from '@/search/components/ResultListItem/ResultListItems';
+import { TotalCount } from '@/search/components/TotalCount/TotalCount';
 
-import { ResultListItem } from '../../components/ResultListItem/ResultListItem';
 import { SearchBar } from '../../components/SearchBar/SearchBar';
-import { useGetSearch } from '../../hooks/useGetSearch';
-import { NoResult } from '../NoResult/NoResult';
+
+import {
+  StyledResultContent,
+  StyledResultContentWrap,
+  StyledResultWrap,
+  StyledSearchWrap,
+  StyledFlexGapContainer,
+  StyledResultListItemsWrap,
+} from './Search.style';
 
 export const Search = () => {
   const [searchParams] = useSearchParams();
   const query = searchParams.get('query');
```

Comment:
```suggestion
  const query = searchParams.get('query') ?? '';
```

`ErrorBoundary` 컴포넌트에 `query` props를 전달할 때 매번 `query || ''`를 선언하는 것보다 처음 `searchParams.get()`을 호출할 때 기본값을 설정해주는게 더 깔끔할 것 같아요!

Reply:
오오 좋습니당 굿굿